etl/requirements.txt

```diff
@@ -1,4 +1,4 @@
 python-dotenv==1.0.0
 pandas==2.2.2
 SQLAlchemy==2.0.32
-psycopg2==2.9.9
\ No newline at end of file
+psycopg2-binary==2.9.9
```

Comment:
On the gcp vm, `psycopg2==2.9.9` installation would still work fine?

Reply:
I've just checked, and it seems to work fine now. I'll go ahead and remove that commit.